document/core/binary/types.rst

```diff
@@ -130,7 +130,9 @@ Limits
    \begin{array}{llclll}
    \production{limits} & \Blimits &::=&
      \hex{00}~~n{:}\Bu32 &\Rightarrow& \{ \LMIN~n, \LMAX~\epsilon \} \\ &&|&
-     \hex{01}~~n{:}\Bu32~~m{:}\Bu32 &\Rightarrow& \{ \LMIN~n, \LMAX~m \} \\
+     \hex{01}~~n{:}\Bu32~~m{:}\Bu32 &\Rightarrow& \{ \LMIN~n, \LMAX~m \} \\ &&|&
+     \hex{04}~~n{:}\Bu64 &\Rightarrow& \{ \LMIN~n, \LMAX~\epsilon \} \\ &&|&
+     \hex{05}~~n{:}\Bu64~~m{:}\Bu64 &\Rightarrow& \{ \LMIN~n, \LMAX~m \}
```

Comment:
Shouldn't all these return a pair of idxtype and limits?

Reply:
Also, the decoding width of the values depends on the index type bit. This does not match what's currently implemented in the interpreter, which always uses the same. Is there any benefit in introducing this dependency?

Reply:
Done